scripts/cdn_publish.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+version="$(git describe --abbrev=0 --tags)"
+mkdir -p cdn-dist/gravity/${version}
+
+cp dist/ui-lib/*.css cdn-dist/gravity/${version}
+cp dist/ui-lib/*.js cdn-dist/gravity/${version}
+
+versionList=""
+for version in $(git tag -l --sort=-committerdate); do
+  versionList+="<li>${version}</li>"
+done
+
+file_contents=$(<./cdn-index.template)
+echo "${file_contents//\{versionlist\}/$versionList}" > cdn-dist/gravity/index.html
+
+~/.local/bin/aws s3 sync cdn-dist s3://${CDN_BUCKET} --delete --region=${PROD_BUCKET_REGION} --exclude "*" --include "*.html" --content-type "text/html; charset=utf-8"
```

Comment:
Won't `--delete` remove any previously published files? If so, I'm don't think that's what we want.

I had imagined this to be cumulative, each new Gravity release would be _added_ here, but all previously published one would remain available.

Reply:
since we are creating version sub directories dynamically, you are probably right.  I'll remove that flag.  good catch